compiler/tests/fixtures/strings.fe

```diff
@@ -9,7 +9,7 @@ contract Foo:
         s5: string100
 
     pub def __init__(s1: string42, a: address, s2: string26, u: u256, s3: string100):
-        emit MyEvent(s2, u, s1, s3, a, "static string", "foo")
+        emit MyEvent(s2, u, s1, s3, a, "static string", string100("foo"))
```

Comment:
This came to my attention after I merged #201 so it sparked this PR to tighten our type checks.